### PR TITLE
Map module bug

### PR DESCRIPTION
**Pull request by prykon**
GUI gets updated upon editing a Geo

### CODE REVIEW

File: dt-mapping/drill-down.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ let rebuild_drill_down = ( response, bindFunction, grid_id )=>{
 
             // create button
             html += `<li><button id="${window.lodash.escape( section.parent )}" type="button" ${disabled ? "disabled" : ""}
-                onclick="DRILLDOWN.get_drill_down( '${window.lodash.escape( bindFunction )}', '${window.lodash.escape( section.selected )}' )"
+                onclick="DRILLDOWN.get_drill_down( '${window.lodash.escape( bindFunction )}', '${window.lodash.escape( section.selected )}', false )"
```
Comment (corsacca) [inline dt-mapping/drill-down.js:27]: @prykon Is this one called from the admin?

Comment (prykon) [reply dt-mapping/drill-down.js:27]: Yes; it's the breadcrumb that shows the selected geo buttons.

Comment (corsacca) [reply dt-mapping/drill-down.js:27]: I would expect that enabling caching here would produce the opposite effect, show old names

Comment (prykon) [reply dt-mapping/drill-down.js:27]: I double checked it and the breadcrumb updates on edit if this line is set to true or false. Your call :)

Comment (corsacca) [reply dt-mapping/drill-down.js:27]: The metrics use this function too, so if the impact is the same i'd prefer not changing the functionality

**Comment by corsacca** [comment]
Thanks Dario!

**Pull request merged**